docs: add x509 for faq (#533)
kubernetes-sigs/metrics-server: FAQ.md

## Known issues

> #### Unable to authenticate the request due to an error: x509: certificate signed by unknown authority
> 

Reviewer: ```suggestion
#### Incorrectly configured front-proxy certificate

```
Metrics Server needs to validate requests coming from kube-apiserver. You can recognize problems with front-proxy certificates if you observe line below in your logs:

```
E0524 01:37:36.055326       1 authentication.go:65] Unable to authenticate the request due to an error: x509: certificate signed by unknown authority
```

> Because Metrics Server is an aggregated server for kube-apiserver. So as an aggregated server need sets `--requestheader-client-ca-file` for validate request which send by kube-apiserver. [headrequest(front-proxy)](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authenticating-proxy)

Reviewer: ```suggestion
To fix this problem you need to provide kube-apiserver proxy-client CA to Metrics Server under `--requestheader-client-ca-file` flag. You can read more about this flag in [Authenticating Proxy](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authenticating-proxy)
```

For cluster created by kubeadm:

> 1. Check kube-apiserver front-proxy args(/etc/kubernetes/manifests/kube-apiserver.yaml)

Reviewer: ```suggestion
1. Find your front-proxy certificates by checking arguments passed in kube-apiserver config (by default located in /etc/kubernetes/manifests/kube-apiserver.yaml)
```

```
- --proxy-client-cert-file=/etc/kubernetes/pki/front-proxy-client.crt
- --proxy-client-key-file=/etc/kubernetes/pki/front-proxy-client.key
```

> 2. Prepare front-proxy-ca for Metrics Server

Reviewer: ```suggestion
2. Create configmap including `front-proxy-ca.crt`
```

```

> kubectl -nkube-system create configmap front-proxy-ca --from-file=front-proxy-ca.crt=/etc/kubernetes/pki/front-proxy-ca.crt -o yaml | kubectl -nkube-system replace configmap front-proxy-ca -f -

Reviewer: Have you considered storing this file in `secret` instead of `configmap`? I'm just worried that some users will upload key by mistake.

Author: It would be a bit embarrassing...

```

> 3. Configure your Metrics Server deployment:

Reviewer: ```suggestion
3. Mount configmap in Metrics Server deployment and add `--requestheader-client-ca-file` flag
```

> ```

Reviewer: Have you considered using `kubectl patch` instead?

Author: I tried, but it seems that path cannot modify args.
